content/chat/typing.textile

```diff
@@ -1,7 +1,91 @@
 ---
 title: Typing indicator
-meta_description: "Real time functionality to indicator when a client is typing or has stopped typing."
+meta_description: "Realtime functionality to indicate when a member is typing or has stopped typing."
 product: chat
 languages:
   - javascript
 ---
+
+Signal when a member is typing to indicate to other members that a message is being written.
+
+<img src="/images/products/chat/typing.png" alt="Example of a chat message input box, above this has two names of other members who are currently typing.">
+
+h2(#implementation). Implementation
+
+Use the "presence":/presence-occupancy/presence feature to implement a typing indicator, this feature would only be possible with all members "entered":/chat/online-status#enter already into the presence set.
```

Comment:
This is a tad jumbled up and unclear. I'll have a go at re-jigging it, but I lack some of your understanding, so forgive me if it's not 100% (or 80% 😜).